_targets.R

```diff
@@ -78,6 +78,19 @@ list(
   #   }
   # ),
 
+  # get the test data for each site
+  tar_target(
+    incoming_data_csvs_upload,
+    {
+      start_dt = "2023-11-30 07:00:00"
+      end_dt = "2023-11-30 10:00:00"
```

Comment:
Thought: I realize this is to have test data to play with but in practice next year, I am wondering if we should have start_dt be a day or two before sys.date in case we upload old logs (ie vulink not transimitting data but still logging)

Reply:
How we decide what date times to pull from the hydrovu api will be from some combination of detecting when the last flagged data was and when the last API pull was (right now it is only based on when the last flagged data was). When we upload old logs it will detect when the last flagged date/api pull date was and it will set that as the start date. I think that answers this question?